src/scripts/glue_jobs/dynamic_record_transformation.py

```diff
@@ -0,0 +1,41 @@
+"""
+DynamicFrame : A Dynamic Frame is similar to an Apache Spark dataframe, which is a data abstraction used to
+organize data into rows and columns, except that each record is self-describing so no schema is required initially.
+see link: https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-crawler-pyspark-extensions-dynamic-frame.html
+
+DynamicRecord : a DynamicRecord represents a logical record within a DynamicFrame. It is like a row in a Spark
+DataFrame. see link: https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-crawler-pyspark-extensions-dynamic-frame.html
+
+This script is used to perform transformations on dynamic record, such as computing value for a column.
+
+"""
+
+from utils import *
+
+DOWNLOADED_FILE_HANDLE_ID = "downloaded_file_handle_id"
+REQUESTED_FILE_HANDLE_ID = "requested_file_handle_id"
+RECORD_DATE = "record_date"
+ASSOCIATED_OBJECT_ID = "association_object_id"
+
+def file_download_record_transformation(dynamic_record):
```

Comment:
I do not see the need to separate this logic into a shared script, this is the logic that belongs to the single scripot. If there is common patterns among scripts we can externalize those in utility scripts.